lib/rspec_profiling/collectors/statsd.rb

```diff
@@ -66,6 +66,7 @@ def initialize
         RspecProfiling.config.statsd_host ||= '127.0.0.1'
         RspecProfiling.config.statsd_port ||= '8125'
         RspecProfiling.config.statsd_max_depth ||= 0
+        RspecProfiling.config.statsd_protocol ||= :udp
```

Comment:
Don't we want `:tcp` here? or are you passing it on the WorkloadWisdom side?

Reply:
The constructor defaults to UDP.  My thought was that we'd add this to the Staas configuration when we bump to use v14.